src/js/utils/dom.js

```diff
@@ -209,3 +209,11 @@ export function nextSibling(element) {
 export function previousSibling(element) {
     return element.previousElementSibling;
 }
+
+export function open(link, target, additionalOptions = {}) {
+    let a = document.createElement('a');
+    a.href = link;
+    a.target = target;
+    a = Object.assign(a, additionalOptions);
+    a.click();
```

Comment:
are there any potential browser security features that would block this click vs. a `window.open`?

Reply:
Nope! The only issues that prevent click are on really old versions of safari we don't officially support:

https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/click

Reply:
👍